2024/wg-fedid.html

```diff
@@ -150,18 +150,23 @@
     <div id="background" class="background">
       <h2>Motivation and Background</h2>
       <p>
-        Identity on the Web is critical to online interaction, privacy, and security. W3C fosters an ecosystem where privacy, security, and user sovereignty are all considered. That includes developing new mechanisms for individuals to have the ability to select the identity information, such as assertions, specific credentials, or specific attributes, relevant to a given interaction. These mechanisms must also be viable for the issuers, verifiers, identity providers, and relying parties to exchange information in a secure and privacy-preserving manner. 
+        Identity on the Web is critical to online interaction, with many privacy and security implications. 
       <p>
       <p>
-        The user agent is the coordinator for these transactions. So, while the request and response protocols are being developed elsewhere (e.g., ISO, IETF, OpenID, and other W3C groups), the web platform layer must also be standardized to provide the privacy and security API framework in a protocol-agnostic and formats-agnostic fashion in a manner that is compatible with identity request/response protocols and different formats.
+        The W3C fosters an ecosystem that addresses privacy, security, and user sovereignty.
       </p>
       <p>
-        The group would like to: 
+        This includes developing new mechanisms that allow individuals to select identity information relevant to a given interaction, such as assertions, specific credentials, or specific attributes, supporting an open wallet ecosystem.
+      <p>
+      <p>
+        These mechanisms must also be viable for issuers, identity providers, verifiers, and relying parties to exchange information securely and privacy-preserving.
```

Comment:
```suggestion
        These mechanisms must also be viable for issuers, identity providers, verifiers, and relying parties to exchange information securely and as privately as possible.
```

Just fixing grammar. Or could say "in a privacy-preserving manor"